src/_data/side-nav.yml

```diff
@@ -37,11 +37,12 @@
       permalink: /dart-vm
 
 - title: Testing
-  children:
-  - title: Dart Testing
-    permalink: /guides/testing
+  permalink: /guides/testing
+  urlExactMatch: true
 
 - title: Resources
+  permalink: /resources
```

Comment:
This page is pretty useless. Would it be a problem to remove the permalink? Another option is to make the page actually, you know, useful. At the very least, it should include links to everything listed under it in the sidenav.